DependencyInjection/Compiler/WellKnownSchemaFilterPass.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\DependencyInjection\Compiler;
+
+use Doctrine\DBAL\Configuration;
+use Symfony\Component\Cache\Adapter\PdoAdapter;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler;
+use Symfony\Component\Lock\Store\PdoStore;
+use Symfony\Component\Messenger\Transport\Doctrine\Connection;
+
+/**
+ * Blacklist tables used by well-known Symfony classes.
+ */
+class WellKnownSchemaFilterPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (! method_exists(Configuration::class, 'setSchemaAssetsFilter')) {
+            // only supported when using doctrine/dbal 2.9 or higher
+            return;
+        }
+
+        $blacklist = [];
+
+        foreach ($container->getDefinitions() as $id => $definition) {
```

Comment:
Looks like `$id` is never used, let's remove it.

```suggestion
        foreach ($container->getDefinitions() as $id => $definition) {
```

Reply:
removed